### DISCUSSION TITLE
InfluxDB 2

### DISCUSSION THREAD

**Discussion by mirkocomparetti-synesis** [pierosavi/pierosavi-imageit-panel#73] [Q&A]
Hello,
Has anyone tried to use influxdb and the new panel 1.0.1?
I managed to make it work with influx v1, but with influxv2 I'm having an hard time to find the proper ID to use to retrieve the sensor reading, as aliasing does not really exist anymore.

Thanks!

**Comment by pierosavi** [comment]
Check this https://github.com/pierosavi/pierosavi-imageit-panel/issues/74#issuecomment-808730711 :)
